kinematic_properties.py

```diff
@@ -242,24 +240,77 @@ def get_inertia_tensor(
        Masses of the particles.
      - position: unyt.unyt_array
        Positions of the particles.
+     - radius: unyt.unyt_quantity
+       Exclude particles outside this radius for the inertia tensor calculation
+     - reduced: bool
+       Whether to calculate the reduced inertia tensor
+     - max_iterations: int
+       The maximum number of iterations to repeat the inertia tensor calculation
 
     Returns the inertia tensor.
-    """
-    # 3x3 inertia tensor
-    Itensor = (
-        mass[:, None, None] / mass.sum() * position[:, None:, None] * position[:, None]
-    ).sum(axis=0)
-
-    # Symmetric, so only return lower triangle
-    Itensor = np.concatenate([np.diag(Itensor), Itensor[np.triu_indices(3, 1)]])
-
-    return Itensor
-
-
-def get_projected_inertia_tensor(
-    mass: unyt.unyt_array, position: unyt.unyt_array, axis: int
-) -> unyt.unyt_array:
-    """
+    '''
+    # Shouldn't calculate an inertia tensor for small numbers of particles
```

Comment:
This condition exists to handle cases where the halo may have more than 100 particles but less than 20 within a SO radius, right? If so, mention this to make a distinction from the 100 particle limit imposed at the subhalo level.

Reply:
I've updated the particle description in https://github.com/SWIFTSIM/SOAP/pull/82/commits/25bd4fc9b8998e9e51c14f80fda40dcb076f8ea7